src/Ansatz/Chain.jl

```diff
@@ -187,6 +187,33 @@ function canonize_site!(::Open, tn::Chain, site::Site; direction::Symbol, mode =
     return tn
 end
 
```

Comment:
```suggestion

truncate(tn::Chain, bond; kwargs...) = truncate!(deepcopy(tn), bond; kwargs...)
truncate!(tn::Chain, bond; kwargs...) = truncate!(boundary(tn), tn, bond; kwargs...)

```

Reply:
Why do we need to dispatch on `boundary`?

Reply:
Oops, that was a mistake. I remove that, sorry!